client/lib/analytics/ad-tracking/woo/record-gtm-datalayer-event.ts

```diff
@@ -0,0 +1,38 @@
+import { mayWeTrackByTracker } from '../../tracker-buckets';
+import { debug, TRACKING_IDS } from '../constants';
+import { initGTMContainer, loadGTMContainer } from '../gtm-container';
+
+/**
+ * Sends a step event to Google Tag Manager for the woo express stepper flow.
+ *
+ * @param {string} stepName
+ * @returns {void}
+ */
+export default async function recordGTMDatalayerEvent( stepName = '' ) {
+	if ( stepName ) {
```

Comment:
as long as we set stepName as required in the type definitions, then there's no need to check it here IMO